app/data/1_0205.json

```diff
@@ -90,7 +104,14 @@
                                 "label": "What was the value of the business's total sales of household goods?",
                                 "guidance": "<h4>Include</h4><ul> <li>VAT</li> <li>internet sales</li> <li>carpets, rugs and other floor coverings</li> <li>furniture</li> <li>household textiles and soft furnishings</li> <li>prints and picture frames</li> <li>antiques and works of art</li> <li>domestic electrical appliance, audio/visual equipment and home computers</li> <li>lighting and minor electrical supplies</li> <li>records, compact discs, audio and video tapes</li> <li>musical instruments and goods</li> <li>decorators’ and DIY supplies</li> <li>lawn-mowers</li> <li>hardware</li> <li>china, glassware and cutlery</li> <li>novelties, souvenirs and gifts</li> </ul>",
                                 "type": "Currency",
-                                "mandatory": false
+                                "mandatory": false,
+                                "validation": {
+                                  "messages": {
+                                    "MANDATORY": "Please provide a value, even if your value is 0",
```

Comment:
The questions are mandatory so so we need the mandatory message? 

Just means there's no way for me to test it?